packages/@lwc/integration-karma/package.json

```diff
@@ -4,6 +4,7 @@
     "version": "2.32.1",
     "scripts": {
         "start": "karma start ./scripts/karma-configs/test/local.js",
+        "start:native": "DISABLE_SYNTHETIC=1 yarn start",
```

Comment:
Rather than adding a new script here, I'd recommend removing this line entirely:

https://github.com/salesforce/lwc/blob/02305bf7afcf2e48a864e794627618cc20f600ed/packages/%40lwc/integration-karma/package.json#L10

Yes it is convenient when testing locally, but we don't actually use it anywhere, so it just adds more noise to this file. Plus we don't have equivalent shorthands for all our other various env vars.